101-backup-la-reporting/azuredeploy.json

```diff
@@ -0,0 +1,176 @@
+    "contentVersion":"1.0.0.0",
+    "parameters":{
+      "workspaceLocation": {
+         "type": "string",
+         "metadata": {
+           "description": "Specify the workspace region"
+         }
+       },
+      
+       "workspaceName":{
+          "type":"string",
+          "metadata":{
+             "description":"Specify the workspace name"
+          }
+       },
+       "_artifactsLocation":{
+          "type":"string",
+          "defaultValue":"https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-backup-la-reporting/",
+          "metadata":{
+             "description":"The base URI where artifacts required by this template are located"
+          }
+       },
+       "_artifactsLocationSasToken":{
+          "type":"securestring",
+          "defaultValue":"",
+          "metadata":{
+             "description":"The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated"
+          }
+       }
+    },
+    "variables":{
+       "nestedTemplates":{
+          "backup_jobs_non_log":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/backup_jobs_non_log.json', parameters('_artifactsLocationSasToken'))]",
+          "backup_jobs_log":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/backup_jobs_log.json', parameters('_artifactsLocationSasToken'))]",
+          "restore_jobs":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/restore_jobs.json', parameters('_artifactsLocationSasToken'))]",
+          "azure_alerts":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/azure_alerts.json', parameters('_artifactsLocationSasToken'))]",
+          "on_prem_alerts":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/on_prem_alerts.json', parameters('_artifactsLocationSasToken'))]",
+          "backup_items":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/backup_items.json', parameters('_artifactsLocationSasToken'))]",
+          "cloud_storage":"[concat(parameters('_artifactsLocation'), 'nestedtemplates/cloud_storage.json', parameters('_artifactsLocationSasToken'))]"
+       }
+    },
+
+          "resources":[
+             {
+                "apiVersion":"2019-05-01",
+                "name": "backup_jobs_non_log",
+                "type":"Microsoft.Resources/deployments",
+                "properties":{
+                   "mode":"Incremental",
+                   "templateLink":{
+                      "uri":"[variables('nestedTemplates').backup_jobs_non_log]",
+                      "contentVersion":"1.0.0.0"
+                   },
+                   "parameters":{
+                     "workspaceLocation":{"value": "[parameters('workspaceLocation')]"} ,
+                     "workspaceName":{"value": "[parameters('workspaceName')]"} 
+                  
+                  }
+                   
+                  
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
Updated